### ISSUE TITLE
explosions destroy fuel tank contents

### ISSUE THREAD

**Issue by Slava0135** [space-wizards/space-station-14#17821]
## Description

If tank explodes you will expect that contents will spill on ground but this is not the case.

**Reproduction**

1. Throw bomb at fuel tank
2. Tank is gone
3. Fuel is gone too :(

**Screenshots**

![Peek 2023-07-04 19-37](https://github.com/space-wizards/space-station-14/assets/40753025/8ff00b2e-43e0-4137-831a-11acf9087e1b)

**Additional context**

**Comment by deltanedas** [comment]
shouldnt it chain explode the fuel tank

also fuel tanks imo shouldnt leave fuel behind after exploding, what exploded???
water should spill tho

**Comment by Matz05** [comment]
Yeah, they should chain explode, but they currently just... disappear.

**Comment by Slava0135** [comment]
> Yeah, they should chain explode, but they currently just... disappear.

They do chain explode but only if damage is low (e.g. potassium explosion or explosive banana peel)

**Comment by deltanedas** [comment]
if water tank blew up from fire it should boil into vapour

**Comment by kontakt** [comment]
Currently still an issue, a hand grenade will delete a fuel tank while a banana bomb will detonate the fuel tanks resulting in a significantly more explosive result.
